hack/ensure-go.sh

```diff
@@ -31,7 +31,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(go version)"
   local minimum_go_version
-  minimum_go_version=go1.20.0
+  minimum_go_version=go1.20.5
```

Comment:
I see the triage-party image is using 1.20.6; do we want the two values to match?

Reply:
I can change triage-party image version